impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -23,10 +23,9 @@ static TextureGLES::Type GetTextureTypeFromDescriptor(
   const auto render_target =
       static_cast<TextureUsageMask>(TextureUsage::kRenderTarget);
   const auto is_msaa = desc.sample_count == SampleCount::kCount4;
-  if (usage == render_target && !is_msaa) {
-    // TODO(matanlurey): MSAA render buffers?
-    // See https://github.com/flutter/flutter/issues/137095.
-    return TextureGLES::Type::kRenderBuffer;
+  if (usage == render_target) {
+    return is_msaa ? TextureGLES::Type::kRenderBufferMultisampled
+                   : TextureGLES::Type::kRenderBuffer;
```

Comment:
This should something like usage == render_target && desc.pixelformat == PixelFormat::S8

Reply:
or maybe pixelformat != PixelFormat::whatever_the_gl_pixel_format_is

Reply:
or maybe even better just pass in the expected attachment point and use that.

Reply:
Unfortuantely the TextureGLES is created far before the concept of attachment points :-/

Reply:
RIP. The formats for GLES are more or less constant, though if that changes this will be a fun landmine 😆